src/Sarif.Viewer.VisualStudio/Sarif/Result.extensions.cs

```diff
@@ -2,6 +2,9 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information. 
 
 using System;
```

Comment:
>System [](start = 6, length = 6)

Can you please get these files renamed properly to Result.Extensions.cs, etc., in your next change?

Reply:
You're looking for just the case change 'e' => 'E'? Sure.